content/using-sagemaker-managed-spot-training/cleanup.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Cleanup"
+date: 2018-08-07T08:30:11-07:00
+weight: 30
+---
+
+Each notebook contains a section at the bottom for cleaning up any resources that were created during the execution of the workbook. All other resources, including the Amazon SageMaker Notebook Instance and VPC can be deleted by deleting the CloudFormation Stack you deployed at the start of this lab.
```

Comment:
One for your consideration: This is probably another of the sections that perhaps may benefit from a {{% notice warning %}} for highlighting the importance.